libs/api/domains/official-journal-of-iceland-application/src/lib/ojoiApplication.service.ts

```diff
@@ -0,0 +1,29 @@
+import { OfficialJournalOfIcelandApplicationClientService } from '@island.is/clients/official-journal-of-iceland/application'
+import { Injectable } from '@nestjs/common'
+import { PostCommentInput } from '../models/postComment.input'
+import { PostApplicationInput } from '../models/postApplication.input'
+import { GetCommentsInput } from '../models/getComments.input'
+
+@Injectable()
+export class OfficialJournalOfIcelandApplicationService {
```

Comment:
What it is the nature of the data in the methods below? Here is not user authentication used to make sure to only read and write data that the user owns?

Reply:
Auth is yet to be implemented on the otherside, will be added in the future.